docs/configuration/configuration.md

```diff
@@ -2797,6 +2797,12 @@ write_relabel_configs:
 # For the `io.prometheus.write.v2.Request` message, this option is noop (always true).
 [ send_native_histograms: <boolean> | default = false ]
 
+# When enabled, remote-write will resolve the URL host name via DNS, choose one of the IP addresses at random, and connect to it. 
```

Comment:
Maybe add: "This is useful when the remote-write receiver uses round-robin DNS."